ws-style.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.bindir = 'exe'
   s.executables = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ['lib']
-  s.required_ruby_version = '>= 2.7.2'
+  s.required_ruby_version = '>= 3.2.0'
```

Comment:
You should leave this at 2.7 for now, otherwise it will prevent anyone still on Ruby 2.7 from getting any updates.